synchronizer/synchronizer.go

```diff
@@ -236,6 +230,102 @@ func (s *ClientSynchronizer) syncBlocks(lastEthBlockSynced *state.Block) (*state
 	return lastEthBlockSynced, nil
 }
 
+// syncTrustedState synchronizes information from the trusted sequencer
+// related to the trusted state when the node has all the information from
+// l1 synchronized
+func (s *ClientSynchronizer) syncTrustedState(latestSyncedBatch uint64) error {
+	if s.IsTrustedSequencer {
+		return nil
+	}
+
+	log.Debug("Getting broadcast URI")
+	broadcastURI, err := s.getBroadcastURI()
+	if err != nil {
+		log.Errorf("error getting broadcast URI. Error: %v", err)
+		return err
+	}
+	log.Debug("broadcastURI ", broadcastURI)
+
+	if broadcastURI != "zkevm-broadcast:61090" {
+		log.Error("URI IS DIFFERENT")
+		log.Error([]byte(broadcastURI))
+		log.Error([]byte("zkevm-broadcast:61090"))
+	}
```

Comment:
I don't understand why we need this. We have a hardcoded url and as we discussed it is a bad idea

Reply:
yeah, looks like you left this while debugging :sweat_smile: